dot/state/service.go

```diff
@@ -233,7 +233,7 @@ func (s *Service) Rewind(toBlock uint) error {
 		}
 	}
 
-	//return s.Base.StoreBestBlockHash(newHead)
+	// return s.Base.StoreBestBlockHash(newHead)
```

Comment:
remove this comment?

Reply:
This is not related with this PR but the linter complains due the missing space

Reply:
> This is not related with this PR

That's okay. It looks like someone forgot to remove that line and we merged it by mistake. It's better you remove it rather than someone creating another PR for removing that line.

Reply:
Removed! thks!